src/pages/User/components/PullRequests/index.js

```diff
@@ -126,6 +126,30 @@ export default class PullRequests extends Component {
     return count;
   }
 
+  getValidPullRequest(data) {
+    let validPullRequest = [];
+    data.items.forEach((pullRequest, index) => {
+      if (!pullRequest.labels.length) {
+        validPullRequest = [...validPullRequest, pullRequest]
```

Comment:
This could be simply changed to:
```suggestion
        validPullRequest.push(pullRequest)
```